src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -836,9 +837,8 @@ pbs_python_setup_resv_class_attributes(void)
 	else
 		DEBUG2_ARG1("BEGIN setting up all reservation attributes %s", "");
 	py_resv_attr_types = PyMem_New(PyObject *, num_entry);
-	memset(py_resv_attr_types, 0, sizeof(PyObject *)*num_entry);
-
 	if (!py_resv_attr_types) { goto ERROR_EXIT;}
+	memset(py_resv_attr_types, 0, sizeof(PyObject *)*num_entry);
```

Comment:
Please put spaces around the '*' between the sizeof*() and num_entry (same for the rest of the memset() calls)

Reply:
Good point. They look like dereferences without the spaces.

Reply:
@bhroam and @mike0042 Thanks fore review. I have addressed your comment. Please have a look into latest commit.